pacta_data_qa.Rmd

```diff
@@ -0,0 +1,660 @@
+
+```{r}
+investors_overview_asset <- result_data_qa %>% 
+  group_by(investor_name, asset_type) %>% 
+  summarise(total_holdings_by_inv_asset = n(),
+            value_usd_by_inv_asset = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup() %>% 
+  group_by(investor_name) %>% 
+  mutate(weight_usd_asset_by_inv = round(value_usd_by_inv_asset/sum(value_usd_by_inv_asset, na.rm = T),4)) %>% 
+  ungroup()
+datatable(investors_overview_asset, caption = "Overall # of holdings and value of the input by Investor and asset type")
+```
+
+The table below shows the number of holdings and aggregated value in USD per flag by investor.
+It also provides the weight of each type of flag per investor, which can help get a quick overview of which issues might be most relevant in a given project / for a given investor.
+
+```{r}
+investors_overview_flag <- result_data_qa %>% 
+  group_by(investor_name, flag) %>% 
+  summarise(holdings_by_flag_inv = n(),
+            value_usd_by_flag_inv = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup() %>% 
+  inner_join(investors_overview, by = ("investor_name")) %>% 
+  group_by(investor_name) %>% 
+  mutate(weight_holdings_per_flag_by_investor = round(holdings_by_flag_inv/total_holdings_by_inv, 4),
+         weight_value_per_flag_by_investor = round(value_usd_by_flag_inv/sum_value_usd_by_inv, 4)) %>% 
+  ungroup() %>% 
+  select(-c(total_holdings_by_inv,sum_value_usd_by_inv))
+# add share value investor
+datatable(investors_overview_flag, caption = "Overall # of holdings and value of the input by Investor and flag")
+```
+
+### Total holdings and value per investor and portfolio
+Number of holdings and aggregated value in USD of the overall portfolio input - grouped by investor and portfolio
+
+* **total_holdings_inv_pf** is the number of holdings per investor and portfolio
+* **value_usd_inv_pf** is the aggregated value of the holdings in USD by investor and portfolio
+* **weight_value_pf_overall** This  shows the relative value of a given portfolio compared to the overall value across all portfolios analysed. Measured in USD.
+* **weight_value_pf_by_inv** This  shows the relative value of a given portfolio compared to the overall value of the corresponding investor's portfolios. Measured in USD. Insofar, it is a measure of the portfolio's importance for the investor at hand.
+
+
+
+```{r}
+investors_portfolio_overview <- result_data_qa %>% 
+  filter(investor_name != "Meta Investor") %>% 
+  group_by(investor_name, portfolio_name) %>% 
+  summarise(total_holdings_inv_pf = n(),
+            value_usd_inv_pf = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup() %>% 
+  mutate(weight_value_pf_overall = round(value_usd_inv_pf / sum(value_usd_inv_pf), 3)) %>% 
+  group_by(investor_name) %>% 
+  mutate(weight_value_pf_by_inv = round(value_usd_inv_pf / sum(value_usd_inv_pf), 3)) %>% 
```

Comment:
consider if it's important to have `na.rm = TRUE` in these `sum()` commands... maybe not since you're creating `value_usd_inv_pf` just above, but just a thought